docs/parameters.md

```diff
@@ -18,6 +18,7 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | "inodeSize"                  |                                                    |         | The inode size to use when formatting the underlying filesystem. Only supported on linux nodes and with fstype `ext2`, `ext3`, `ext4`, or `xfs`.                                                                                                                                                                                                                                               |
 | "bytesPerINode"              |                                                    |         | The `bytes-per-inode` to use when formatting the underlying filesystem. Only supported on linux nodes and with fstype `ext2`, `ext3`, `ext4`.                                                                                                                                                                                                                                                  |
 | "numberOfINodes"             |                                                    |         | The `number-of-inodes` to use when formatting the underlying filesystem. Only supported on linux nodes and with fstype `ext2`, `ext3`, `ext4`.                                                                                                                                                                                                                                                 |
+| "bigAllocClusterSize         |                                                    |         | The cluster size (in bytes) to use when formatting an `ext4` filesystem with the `bigalloc` option enabled. Specifying a value will enable the `bigalloc` formatting option.                                                                                                                                                                                                                                                                                                |
```

Comment:
From my understanding (and based on the kernel wiki here: https://ext4.wiki.kernel.org/index.php/Bigalloc) `bigalloc` is just the parameter to enable clustered allocation on `ext4`.

Thus, I propose this parameter (and associated internal variable/function names) is renamed to `'clusterSize`  because that is what this is actually doing. This also would make sense if we add support in the future for another filesystem that supports specifying cluster size.

Reply:
I disagree because I think bigAllocClusterSize helps the customer more easily look up what this parameter does. My only grievance with `clusterSize` is that Windows renamed block size to cluster size. This is why I added the explicit mention of bigalloc in. https://support.microsoft.com/en-us/topic/default-cluster-size-for-ntfs-fat-and-exfat-9772e6f1-e31a-00d7-e18f-73169155af95

This issue is exacerbated by the first link coming up when you search "cluster size" is that windows document. 
<img width="1352" alt="Screenshot 2023-08-04 at 2 35 56 PM" src="https://github.com/kubernetes-sigs/aws-ebs-csi-driver/assets/68304519/43e9c8db-fc10-4718-9c55-e02c51329eb6">

Reply:
TLDR: Yes this parameter name SHOULD be `clusterSize`, but because Windows overloaded the term I think we need the explicit mention of ext4's bigAlloc formatting option.

Reply:
In that case, I propose the name `ext4ClusterSize`

Reply:
Suggest `ext4ClusterSize` as proposed above.